src/eHealth.locations/services/selected-location-factory.js

```diff
@@ -246,8 +246,19 @@ angular.module('eHealth.locations.services')
             }
           });
           return angular.copy(adminDivisions);
+        },
+        clear: function () {
+          levels.forEach(function (level) { delete level.selected });
         }
       };
+
+      Object.defineProperty(location, 'innermost', {
```

Comment:
oh, i see. i find using a getter not a big advantage, i would say that it can even be a bit confusing, but we can keep it if you want, it is not a big deal